README.md

```diff
@@ -25,6 +25,7 @@ I.e.:
 		FunctionReader: handlers.MakeFunctionReader(clientset),
 		ReplicaReader:  handlers.MakeReplicaReader(clientset),
 		ReplicaUpdater: handlers.MakeReplicaUpdater(clientset),
+		IdentiyHandler: handlers.MakeIdentityHandler(clientset),
```

Comment:
In retrospect I'm not sure that `identity` is the right word since that is often used to express claims / authorisation.

Reply:
Does the function `MakeIdentityHandler` need to take a `clientset` given what it is doing?

Reply:
You're right. `clientset` is not required. I was just following the pattern (obviously without thinking about it too much)

I'll remove that so it's an empty call